pyproject.toml

```diff
@@ -43,6 +43,7 @@ python = ">=3.8,<4.0"
 volttron = "^10.0.2rc0"
 openleadr = "0.5.27"
 cryptography = ">=36.0.1,<37.0.0"
+gevent = "^21.12.0"
```

Comment:
This should be installed by volttron right?  We shouldn't need to have the gevent here.

Reply:
Yes, I initially included gevent because I was having problems installing the agent on volttron due to some gevent dependency issues. But that was awhile ago. I just tested reinstalling this agent without including gevent in pyproject and installation succeeds. Will remove.